test/signature_parser.py

```diff
@@ -41,6 +41,18 @@ def has_eigen_compatible_arg(self):
         """Return true if any argument is vector-like (can be an Eigen c++ type)"""
         return any(arg in ("matrix", "vector", "row_vector") for arg in self.stan_args)
 
+    def has_scalar_only_args(self):
```

Comment:
[question]
What is this python program for?

Reply:
The Python test retrieves the list of all stanc3 supported signatures and this function was used to trim that list down to only functions that have scalar arguments. But given that I realized this is most likely overkill, this will go away anyways.